packages/flutter_tools/lib/src/commands/config.dart

```diff
@@ -26,6 +27,16 @@ class ConfigCommand extends FlutterCommand {
       negatable: false,
       hide: !verboseHelp,
       help: 'Print config values as json.');
+    for (Feature feature in allFeatures) {
+      if (feature.configSetting == null) {
+        continue;
+      }
+      argParser.addFlag(
+        feature.configSetting,
+        help: feature.generateHelpMessage(),
+        negatable: true,
+      );
+    }
```

Comment:
Do we want an extra one to help people get back to default / clear all overrides after they messed with them? Everything turned on isn't going to be the default as we add more experimental features.

Reply:
Added a `--clear-features` which removes all feature settings.